### PR TITLE
OCPQE-23157: correct target build arch

### PR DESCRIPTION
**Pull request by jianlinliu**

### CODE REVIEW

**Comment by jianlinliu** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.12-arm64-stable-4.12-upgrade-from-stable-4.11-aws-ipi-proxy-sts-f28

**Comment by jianlinliu** [comment]
The pj-relearse job [log](https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/53578/rehearse-53578-periodic-ci-openshift-openshift-tests-private-release-4.12-arm64-stable-4.12-upgrade-from-stable-4.11-aws-ipi-proxy-sts-f28/1805193332066881536) showed `cucushift-upgrade-setchannel` step got passed.

cc @shellyyang1989 @liangxia to review.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__arm64-nightly-4.12-upgrade-from-stable-4.11.yaml [hunk 1]
```diff
@@ -56,6 +56,7 @@ releases:
       version: "4.12"
   latest:
     prerelease:
+      architecture: amd64
```
Comment (shellyyang1989) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__arm64-nightly-4.12-upgrade-from-stable-4.11.yaml:59]: Thanks Jianlin for the fix! I noticed that we didn't set arch for some amd64 images in https://github.com/openshift/release/pull/53318/files and thought the default arch is amd64. What's the default value if unset? cc @JianLi-RH for awareness

Comment (liangxia) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__arm64-nightly-4.12-upgrade-from-stable-4.11.yaml:59]: I think it defaults to amd64. And define it explicitly makes it more clear.

Comment (jianlinliu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__arm64-nightly-4.12-upgrade-from-stable-4.11.yaml:59]: Yeah, just to make it more clear to read.

**Pull request merged**